tap_linkedin_ads/client.py

```diff
@@ -223,8 +238,9 @@ def check_accounts(self, config):
         factor=2
     )
     def request(self, method, url=None, path=None, **kwargs):
+        self.fetch_and_set_access_token()
         if not self.__verified:
-            self.__verified = self.check_access_token()
+            self.__verified = self.fetch_and_set_access_token()
```

Comment:
`fetch_and_set_access_token` is still not returning anything, so I don't think this would work. I think it needs to return what we want to se `__verified` to, or set `__verified` in the function. 

We also call `fetch_and_set_access_token` just above in L241. Do we want to call this regardless of whether or not it's been verified? If so, I think we could remove the check here. If not, I think we should remove the call at L241. I believe we only want it to call it when we're not verified, but I could be mistaken.